pkg/security/admission/byrestrictions.go

```diff
@@ -1,52 +0,0 @@
-package admission
```

Comment:
I would keep this for backwards compatibility and make it the secondary sort after priority order